### ISSUE TITLE
Failed to load resource: the server responded with a status of 404 ()

### ISSUE THREAD

**Issue by paolov1928** [sugarshin/react-instagram-embed#210]
Really enjoy using the tool, thanks!

Recently I have been encountering a 404 error when using the bike example in this repo. Is anyone else having the same 404?

**Comment by kpennell** [comment]
Yeah, same problem. I had to include "https://www.instagram.com/static/bundles/es6/EmbedSDK.js/47c7ec92d91e.js" on my index.html page for this to work.

**Comment by kpennell** [comment]
This started happening after I upgraded gatsby and other plugins.

**Comment by Coderwelsch** [comment]
@kpennell → that happened also to me, got that error after an `yarn upgrade` on a gatsbyjs project …

**Comment by bethshook** [comment]
I'm also receiving a 404 on a Gatsby project

**Comment by Sajidkhan44** [comment]
> Yeah, same problem. I had to include "https://www.instagram.com/static/bundles/es6/EmbedSDK.js/47c7ec92d91e.js" on my index.html page for this to work.

can u write the syntax so that I can copy and paste that in my code
